docpages/example_programs/the_basics/editing-objects.md

```diff
@@ -0,0 +1,87 @@
+\page editing-objects Editing Channel and Message
+
+Sometimes we need to update an object, such as message or channel. At first, it might seem confusing, but it's actually really simple! It's done with \ref callback-functions callbacks. You just need to use an object with identical properties you don't need to update.
+
+~~~~~~~~~~{.cpp}
```

Comment:
Preview shows this code block as broken. I'm assuming that's because the .md isn't linked correctly. Please leave this until the other changes have been corrected, I'll update this after.

Reply:
Okay, but what code block?

Reply:
The `~~~~~~~~~~` shows as a code block on the website. Please leave this comment alone until you've fixed the other changes as it may not even be valid.

Reply:
Roger that

Reply:
Oh yeah, it shows me that it's a code block, too. Maybe I can fix it since it also says that in the IDE I'm using.

Reply:
Fixed